cmd/influxd/upgrade/database_test.go

```diff
@@ -37,6 +37,7 @@ func TestUpgradeRealDB(t *testing.T) {
 
 	boltPath := filepath.Join(tl.Path, bolt.DefaultFilename)
 	enginePath := filepath.Join(tl.Path, "engine")
+	cqPath := filepath.Join(tl.Path, "cq.txt")
```

Comment:
Would it be worth adding some CQs to the checked-in test data here, so we can test against it?

Reply:
Having tests is the best way to ensure it isn't broken in the future 👍